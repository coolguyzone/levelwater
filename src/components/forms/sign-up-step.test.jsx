import React from 'react';
import { shallow } from 'enzyme';
import { SignUpStep } from './sign-up-step.jsx';
import toJson from 'enzyme-to-json';

describe('sign up step', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<SignUpStep />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
