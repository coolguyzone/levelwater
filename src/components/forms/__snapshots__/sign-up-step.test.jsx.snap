exports[`sign up step renders a snapshot 1`] = `
<div
  className="ui grid"
  id="sign-up-step">
  <div
    className="column thirteen wide computer one wide tablet" />
  <div
    className="column three wide computer five wide tablet">
    Step 
     of 6
  </div>
</div>
`;
